File: codes/java_codes/factorial.java
import java.util.*;
public class factorial {
    public static void main(String[] args){
        Scanner sc = new Scanner(System.in);
        long n,i,fact=1;
        System.out.println("Enter the number for calculating factorial: ");
        n = sc.nextLong();
        for(i=2;i<=n;i++){
            fact*=i;
        }
        System.out.println("Factorial of "+n+" is: "+fact+" !!!");
    }
}

File: codes/python_codes/factorial.py
# -*- coding: utf-8 -*-
"""
Created on Fri Jun  5 17:56:31 2020

@author: ankit
this function calculates factorial
"""
def fact(n):
    if(n<2):
        return 1
    else:
        return n*fact(n-1)


print("Enter the number for factorial:")
n=int(input())
print("Factorial of ",n," is: ",fact(n))

